### ISSUE TITLE
Update hotspot antenna and elevation

### ISSUE THREAD

**Issue by nmharmon8** [helium/hotspot-app#520]
When I go to update hotspot I select "Custom Antenna" enter 8.5 dbi and 10 meters. I then click Update Antenna, but the confermation screen shows "1 dbi" not 8.5 dbi. 

Also if I don't press enter on the keyboard before clicking update antenna the height field is not recorded. 

Given this was not working I selected the RAK antenna as it is the closest at 5.8 dbi. After submitting I used the API to  check what the setting was and it shows 58 for gain. Is this correct. Not representing as a float? See below.

``` json
{"data":{"lng":-104.82144351015835,"lat":39.7211750174995,"timestamp_added":"2020-09-09T23:52:17.000000Z","status":{"online":"online","listen_addrs":["/ip4/98.43.151.233/tcp/49016"],"height":822138},"reward_scale":0.4310302734375,"owner":"13gJqE2X4mjA1VjJ3UvNgSoebMogCiQhBPyQWrGKSXm8ZH6Bnzp","nonce":2,"name":"muscular-mocha-jellyfish","location":"8c268cd0b2621ff","last_poc_challenge":820768,"last_change_block":822834,"geocode":{"short_street":"Sable Blvd","short_state":"CO","short_country":"US","short_city":"Aurora","long_street":"Sable Boulevard","long_state":"Colorado","long_country":"United States","long_city":"Aurora","city_id":"YXVyb3JhY29sb3JhZG91bml0ZWQgc3RhdGVz"},"gain":58,"elevation":30,"block_added":490492,"block":822856,"address":"116GfBi3trAuVxkd3wJmR5Rn2GUnPysYU31v79HvoCwCYscp9x2"}}
```

**Comment by cokes518** [comment]
>  it shows 58 for gain

Yes, that is expected. 

The bug fix for entering antenna and height is in 3.1.1 which is in app review. it'll be out in the next few days.

**Comment by nmharmon8** [comment]
Great. Thanks.
